nethack_server/src/db_pgsql.c

```diff
@@ -446,6 +486,60 @@ db_add_new_game(int uid, const char *filename, const char *role,
     return gid;
 }
 
+void
+db_add_binge_entry(int gid, int level, int hp, int max_hp, int gold, int moves,
+                   int energy, int max_energy, int attrib_str, int attrib_int,
+                   int attrib_wis, int attrib_dex, int attrib_con, int attrib_cha,
+                   int score)
+{
+    PGresult *res;
+    char gid_str[16],
+         level_str[16],
+         hp_str[16],
+         max_hp_str[16],
+         gold_str[16],
+         moves_str[16],
+         energy_str[16],
+         max_energy_str[16],
+         attrib_str_str[16],
+         attrib_int_str[16],
+         attrib_wis_str[16],
+         attrib_dex_str[16],
+         attrib_con_str[16],
+         attrib_cha_str[16],
+         score_str[16];
+
+    const char *const params[] = {gid_str, level_str, hp_str, max_hp_str,
+        gold_str, moves_str, energy_str, max_energy_str, attrib_str_str,
+        attrib_int_str, attrib_wis_str, attrib_dex_str, attrib_con_str,
+        attrib_cha_str, score_str
+    };
+    const int paramFormats[] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
`const int paramFormats[] = { 0 };` is a better way to initialize this to `0`.

http://stackoverflow.com/a/201116